src/OpenFeature.Contrib.Providers.FeatureManagement/OpenFeature.Contrib.Providers.FeatureManagement.csproj

```diff
@@ -3,7 +3,8 @@
   <PropertyGroup>
     <PackageId>OpenFeature.Contrib.Provider.FeatureManagement</PackageId>
     <VersionNumber>0.0.1</VersionNumber>
-    <Version>$(VersionNumber)-preview</Version>
+    <VersionPrefix>$(VersionNumber)</VersionPrefix>
+    <VersionSuffix>preview</VersionSuffix>
```

Comment:
N.B. `<VersionSuffix>` will still be overridden in CI builds to be, e.g., `0.0.1-ci.$(date -u +%Y%m%dT%H%M%S)+sha.${GITHUB_SHA:0:9}` when `--version-suffix ...` is passed, but `preview` will still be used when no `--version-suffix ...` is passed, as in the `release.yml` workflow.